config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -404,7 +404,131 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-master-containerd-serial-slow
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs serial Windows tests on a Kubernetes cluster running containerd provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud
-- interval: 24h
+- interval: 48h
+  name: ci-kubernetes-e2e-azure-disk-windows
+  decorate: true
+  decoration_config:
+    timeout: 2h
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-azure-windows: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes-sigs
+    repo: azuredisk-csi-driver
+    base_ref: master
+    path_alias: sigs.k8s.io/azuredisk-csi-driver
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200326-e946722-1.18
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
+      - --aksengine-creds=$(AZURE_CREDENTIALS)
+      - --aksengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.48.0/aks-engine-v0.48.0-linux-amd64.tar.gz
```

Comment:
We should consider using `https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz` for jobs that are testing against the master branch in case of any upstream changes breaking aks-engine (e.g. `kubectl` incident). We can use the redirection link to quickly deploy a canary build of aks-engine to fix the issue.